twilight-model/src/application/command/option.rs

```diff
@@ -199,6 +208,27 @@ pub enum CommandOptionValue {
     Number(f64),
 }
 
+impl CommandOptionValue {
+    /// This range contains integer values that safely can be
+    /// represented as a 64-bit floating point value.
+    ///
+    /// Values outside of this range will result in a `400 Bad
+    /// Request`.
+    pub const INTEGER_RANGE: Range<i64> =
+        -(2_i64.pow(f64::MANTISSA_DIGITS) - 1)..(2_i64.pow(f64::MANTISSA_DIGITS));
+    /// This range contains integer values that safely can be
```

Comment:
This is technically incorrect, as it's not a range of integers, with respect to type or math.

Reply:
I changed it to
```
    /// Within this floating point range f64 can represent all integer
    /// values exactly, outside this range it might not be possible.
```

Reply:
It's a bit grammatically awkward. How about
```
/// This range contains all floating point values that can be
/// safely used as Discord Number values.
```
?